features/admin.connections.v1/components/edit/settings/idp-certificates/idp-certificates.tsx

```diff
@@ -397,45 +409,63 @@ export const IdpCertificates: FunctionComponent<IdpCertificatesV2Props> = (props
                         </Grid>
                     )
                 }
-                <Grid direction="column" container spacing={ 3 } xs={ 12 }>
-                    { isJWKSEnabled && isPEMEnabled && (
-                        <Grid md={ 12 } lg={ 6 }>
-                            <Switcher
-                                widths={ "two" }
-                                compact
-                                data-testid={ `${ testId }-switcher` }
-                                selectedValue={ selectedConfigurationMode }
-                                onChange={ onSelectionChange }
-                                options={ [
-                                    {
-                                        label: t("authenticationProvider:forms." +
-                                            "certificateSection.certificateEditSwitch.jwks"),
-                                        value: ("jwks" as CertificateConfigurationMode)
-                                    },
-                                    {
-                                        label: t("authenticationProvider:forms." +
-                                            "certificateSection.certificateEditSwitch.pem"),
-                                        value: ("certificates" as CertificateConfigurationMode)
-                                    }
-                                ] }
-                            />
-                        </Grid>
-                    ) }
+                <Grid md={ 12 } lg={ 6 }>
+                    <Heading as="h4">
+                        { t("console:develop.features.authenticationProvider.forms.certificateSection.heading") }
+                    </Heading>
+                </Grid>
+                <Grid md={ 12 } lg={ 6 }>
```

Comment:
In the screenshot, there is a considerable padding between the header and subheading. Shall we reduce the spacing a bit so that header and subheader appear grouped together?